editor/Editor.tsx

```diff
@@ -723,7 +723,7 @@ function Editor(props: RouteComponentProps) {
                         }
                         return (
                             <div
-                                key={v4()}
+                                key={uuid()}
```

Comment:
Just a side observation. I'm guessing the uuid is used here to please eslint-react, but I don't think key-ing with a UUID is desirable. Ideally it would be some piece of data from the array of objects that is deterministic between renders.

Reply:
Thank you for finding this! This should have used the id of the view or track instead. I will update this after testing it.